client/app/Explore/page.js

```diff
@@ -51,82 +51,56 @@ const Explore = () => {
     fetchNFTData();
   }, [nftTransfers]);
 
-  // console.log(nftDataforcard);
-
-  if (loading) {

```

Comment:
Please keep this loading page as it is because it will improve the UX for the user.